### PR TITLE
Arrowhead

### PR DESCRIPTION
**Pull request by mcdaniel67**

### CODE REVIEW

**Comment by mcdaniel67** [comment]
Retrospectively, I should have done each exercise independently instead of building on each exercise. I've switched to doing that starting with DinoDex.

File: arrowhead/arrowhead.rb [hunk 1]
```diff
@@ -2,32 +2,34 @@ class Arrowhead
...
-      if shapes.include? shape
-        arrowhead = shapes[shape]
-        "You have a(n) '#{arrowhead}' arrowhead. Probably priceless."
-      else
-        raise "Unknown shape value. Are you sure you know what you're talking about?"
-      end
-    else
-      raise "Unknown region, please provide a valid region."
-    end
+    region_error unless CLASSIFICATIONS.include? region
```
Comment (djkotowski) [inline arrowhead/arrowhead.rb]: Wrap arguments with parentheses.

File: triangle_facts/triangle_facts.rb [hunk 2]
```diff
@@ -1,58 +1,64 @@
 # Killer facts about triangles AWW YEAH
 class Triangle
```
Comment (djkotowski) [inline triangle_facts/triangle_facts.rb]: Clean up files from other exercises.  Every time you make a branch, make sure you have a clean branch (usually `master`) checked out.

**Comment by djkotowski** [comment]
Nice submission.  Make sure you clean out your branches, though.  As an exercise, remove Triangle Facts from this branch and push the changes.  No need to resubmit a PR though.

**Pull request closed without merging**